README.md

```diff
@@ -0,0 +1,10 @@
+# Homebrew formulas
+
+This repository contains automatically generated [homebrew]
+[formulas] for various Foxygoat tools and CLIs.
+
+[homebrew]: https://brew.sh
```

Comment:
I like to put the link anchors at the end of the section (before the next heading). Up to you though.

Reply:
fixed.